src/Features/CSharp/Portable/CSharpFeaturesResources.resx

```diff
@@ -622,4 +622,7 @@
   <data name="Apply_file_header_preferences" xml:space="preserve">
     <value>Apply file header preferences</value>
   </data>
+  <data name="Explicit_conversion_of_see_cref_T_0_to_see_cref_T_1" xml:space="preserve">
+    <value>Explicit conversion of &lt;see cref="T:{0}"/&gt; to &lt;see cref="T:{1}"/&gt;.</value>
+  </data>
```

Comment:
A better approach here would be `Explicit conversion of {0} to {1}.`.  Then you can form the `<see cref=...` portion in our code itself, and we don't have to worry about localizers *not* touching that portion.